### ISSUE TITLE
Where do I configure the API?

### ISSUE THREAD

**Issue by itsgabraf** [dgreif/ring-alarm#45]
Where do I insert that part under “Setup and Configure” for the API?

**Comment by dgreif** [comment]
https://github.com/dgreif/ring-alarm#setup-and-config is an example of creating a new instance of `RingApi`.  You can also take a look at https://github.com/dgreif/ring-alarm/blob/master/examples/example.ts.  Let me know if you need additional clarification

**Comment by itsgabraf** [comment]
Yea I need a bit more help. I see the code but don’t know where exactly to create it in...

**Comment by dgreif** [comment]
I guess that depends on what you are trying to do. If you have a nodejs project, you would install the @dgreif/ring-alarm package and import it into some file in your project. If you want the homebridge plugin, then you don’t need to interact with the api at all

**Comment by itsgabraf** [comment]
Oh ok. I’m trying to do the HomeKit. The camera says no response though with no images popping up in notifications or the camera preview.

**Comment by dgreif** [comment]
@itsgabraf what type of camera do you have?  Should say in the "Model" field for the camera in the Home app

**Comment by itsgabraf** [comment]
Doorbell V3 - I was able to only once get an image on one device...have tried removing and adding. No luck. And motion isn’t reliable yet either :-/

**Comment by dgreif** [comment]
Unfortunately the doorbell v3 only refreshes it's snapshot image once ever 10 minutes.  To confirm this, open your Ring app and see if you can get that app to refresh the image any faster than 10 minutes (without going into live view mode).  Because of this restriction, the homebridge plugin is not able to pull images very often, which makes it seem like it isn't working.  See previous discussion in #38

**Comment by dgreif** [comment]
Closing in favor of #38
